src/utils/fileUtils.ts

```diff
@@ -0,0 +1,45 @@
+import * as path from "path";
+import { TextDocument, window, workspace, WorkspaceFolder } from "vscode";
+
+/**
+ * Return the current workspace uri from root
+ *
+ * @returns the path from root of the current workspace
+ */
+export function getWorkspaceUri(): string {
+  const currentFile: TextDocument = (window.activeTextEditor && window.activeTextEditor.document && window.activeTextEditor.document.languageId === 'xml') ?  window.activeTextEditor.document : undefined;
+  const currentWorkspace: WorkspaceFolder = (currentFile && currentFile.uri) ?  workspace.getWorkspaceFolder(currentFile.uri) : undefined;
```

Comment:
Please move this currentFile in the function parameter and do like this for other function.